<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrintReservation</class>
 <widget class="QDialog" name="DlgPrintReservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print reservation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="btnPrintConfirmation">
     <property name="text">
      <string>Confirmation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnPrintReservation">
     <property name="text">
      <string>Reservation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnPrintRegistrationCard">
     <property name="text">
      <string>Registration Card</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
